docker/jenkins/Dockerfile.centos8-x86_64

```
@@ -1,4 +1,4 @@
-FROM centos:8
+FROM rockylinux
```

```suggestion
FROM rockylinux/rockylinux:8
```
We want to pin this to the specific version (centos8 equivalent).